opp/flask/json_server.py

```diff
@@ -7,7 +7,7 @@
 #
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
-# Unless required by applicable law or agreed to in writing, software
+# Unless required by applic/able law or agreed to in writing, software
```

Comment:
This file should also be reverted. I suggest copying it over somewhere, reverting, then copying back to continue your work. Soon we need to think about production vs development builds